### DISCUSSION TITLE
Sourcehut not in sync with github

### DISCUSSION THREAD

**Discussion by FabioLolix** [tenacityteam/tenacity-legacy#717] [General]
The sourcehut git repo is not in sync with github git repo, last commit there is [0665614b](https://git.sr.ht/~tenacity/tenacity/commit/0665614b85f1495c75fc1324d7bcead123009e91) of 4 months ago

https://git.sr.ht/~tenacity/tenacity

With this RCF https://github.com/tenacityteam/tenacity/issues/155#issuecomment-877668436 have been declared that Sourcehut would be the primary

There has been some problem or I can migrate the AUR pkgbuild to use github?

**Comment by n0toose** [comment]
Pinging @solfisher.

**Reply by nbsp** [reply]
@Semisol is the one responsible for the syncing, as it's hosted on his VPS
in general I am not affiliated with the project anymore, and I am no longer responsible for the sourcehut infrastructure

**Reply by Semisol** [reply]
I'm no longer participating (usually) and I think the bridge broke, yes.

**Reply by n0toose** [reply]
to be honest i've dropped it as well but i'm like "ok i keep getting notifications about this what to do" lol

whatever, just move it to github
